scaffolding-chef-infra/tests/user-linux-api/habitat/plan.sh

```diff
@@ -9,7 +9,7 @@ pkg_version="1.0.0"
 pkg_scaffolding="ci/scaffolding-chef-infra"
 pkg_svc_user=("root")
 scaffold_policy_name="ci"
-scaffold_chef_client="chef/chef-client/14.13.11"
+scaffold_chef_client="chef/chef-infra-client"
```

Comment:
tests require a version as they parse this line and take the version after the 2nd `/` character to verify the installed version is the same. https://github.com/chef/effortless/blob/main/scaffolding-chef-infra/tests/user-linux-api/tests/test.bats#L1